.github/workflows/test.yml

```diff
@@ -644,6 +644,20 @@ jobs:
           flags: engines,${{ matrix.os }},library,binary
           name: engines-${{ matrix.os }}
 
+      - run: pnpm run test
```

Comment:
Is there a reason we only run this on Linux?

Reply:
I think we just forgot to add it to no-docker below as well @danstarns